### PR TITLE
Release/100.13.0

### PR DESCRIPTION
**Pull request by tanneryould**
Merges changes from 100.13 into `main`. 

Note: I scrolled through each file to verify changes.

### CODE REVIEW

**Comment by tanneryould** [comment]
metadata checker raising false flags

**Review by ldanzinger — APPROVED** [review]
Since you already have reviewed each file, I did a spot check and everything looks good.

**Review by Gela — APPROVED** [review]
Randomly reviewed a handful of files and they looked good. Here's to hoping the other 900 are okay too (Chrome keeps crashing on me due to the size of this PR!)

File: ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/main.cpp [hunk 1]
```diff
@@ -90,3 +90,27 @@ void setAPIKey(const QGuiApplication& app, QString apiKey)
 
   Esri::ArcGISRuntime::ArcGISRuntimeEnvironment::setApiKey(apiKey);
 }
+
+void setAPIKey(const QGuiApplication& app, QString apiKey)
```
Comment (MicahSamaduroff) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/main.cpp:94]: This method already exists, and this causes a compiler error when trying to build the sample

**Review by MicahSamaduroff — CHANGES_REQUESTED** [review]

**Review by MicahSamaduroff — APPROVED** [review]
👍 Looks good to me, I just reviewed that one change and not the whole PR.

**Pull request merged**